function-server/src/main/java/io/dispatchframework/javabaseimage/SimpleFunctionExecutor.java

```diff
@@ -68,13 +78,9 @@ public String execute(String message) {
 
             if (err == null) {
                 try {
-                    r = f.apply(req.getContext(), req.getPayload());
-                } catch (IllegalArgumentException ex) {
-                    err = new Error(ex, ErrorType.INPUT_ERROR);
-                    ex.printStackTrace();
-                } catch (Exception ex) {
-                    err = new Error(ex, ErrorType.FUNCTION_ERROR);
-                    ex.printStackTrace();
+                    r = withTimeout(f, req.getContext(), req.getPayload());
```

Comment:
Looks like last time I forgot we already have `f` as a field, so I guess it's not necessary to pass it as an argument here